### PR TITLE
Remove switchable video

### PR DESCRIPTION
**Pull request by benjaminwp18**
# Pull Request

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Removes video switching from the GUI and adds arm/disarm buttons to the controllers

### CODE REVIEW

**Review by InvincibleRMC — CHANGES_REQUESTED** [review]
Use declare_paremeter. remove threading nonsense

File: src/surface/flight_control/flight_control/manual_control_node.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
 from collections.abc import MutableSequence
+from enum import IntEnum
 
 import rclpy
 from rclpy.executors import MultiThreadedExecutor
 from rclpy.node import Node
+from rclpy.parameter import Parameter
```
Comment (InvincibleRMC) [inline src/surface/flight_control/flight_control/manual_control_node.py:7]:
```suggestion
```

File: src/surface/flight_control/flight_control/manual_control_node.py [hunk 2]
```diff
@@ -38,11 +41,24 @@
...
+class ControllerMode(IntEnum):
+    ARM = 0
+    TOGGLE_CAMERAS = 1
+
 
 class ManualControlNode(Node):
     def __init__(self) -> None:
         super().__init__('manual_control_node')
 
+        mode_param = self.declare_parameter(CONTROLLER_MODE_PARAM, Parameter.Type.INTEGER)
```
Comment (InvincibleRMC) [inline src/surface/flight_control/flight_control/manual_control_node.py:60]:
```suggestion
        mode_param = self.declare_parameter(CONTROLLER_MODE_PARAM)
```

File: src/surface/flight_control/flight_control/manual_control_node.py [hunk 3]
```diff
@@ -70,12 +86,20 @@ def __init__(self) -> None:
             qos_profile_system_default
         )
 
-        # Cameras
-        self.camera_toggle_publisher = self.create_publisher(
-            CameraControllerSwitch,
-            "camera_switch",
-            qos_profile_system_default
-        )
+        controller_mode = ControllerMode(mode_param.get_parameter_value().integer_value)
```
Comment (InvincibleRMC) [inline src/surface/flight_control/flight_control/manual_control_node.py:89]:
```suggestion
        controller_mode = ControllerMode(mode_param.value)
```

File: src/surface/flight_control/flight_control/manual_control_node.py [hunk 4]
```diff
@@ -136,7 +160,7 @@ def valve_manip_callback(self, msg: Joy) -> None:
             self.valve_manip.publish(ValveManip(active=False))
             self.valve_manip_state = False
 
-    def camera_toggle(self, msg: Joy) -> None:
+    def toggle_cameras(self, msg: Joy) -> None:
```
Comment (InvincibleRMC) [inline src/surface/flight_control/flight_control/manual_control_node.py:163]:
```suggestion
    def camera_toggle(self, msg: Joy) -> None:
```

**Review by InvincibleRMC — CHANGES_REQUESTED** [review]

File: src/surface/flight_control/flight_control/manual_control_node.py (hunk 4 above)
Comment (benjaminwp18) [reply src/surface/flight_control/flight_control/manual_control_node.py:163]: Functions should start with verbs
I call "eat_walrus()" on my Walrus-eating robot, not "walrus_eat()"

File: src/surface/flight_control/flight_control/manual_control_node.py [hunk 5]
```diff
@@ -70,12 +85,20 @@ def __init__(self) -> None:
...
 
-        # Cameras
-        self.camera_toggle_publisher = self.create_publisher(
-            CameraControllerSwitch,
-            "camera_switch",
-            qos_profile_system_default
-        )
+        controller_mode = ControllerMode(mode_param.value)
+
+        if controller_mode == ControllerMode.TOGGLE_CAMERAS:
```
Comment (InvincibleRMC) [inline src/surface/flight_control/flight_control/manual_control_node.py:90]:
```suggestion
        if controller_mode is ControllerMode.TOGGLE_CAMERAS:
```

**Review by InvincibleRMC — CHANGES_REQUESTED** [review]

**Review by InvincibleRMC — APPROVED** [review]

**Pull request merged**